gemrb/plugins/GUIScript/GUIScript.cpp

```diff
@@ -8737,6 +8757,8 @@ static PyObject* GemRB_GetSlotItem(PyObject * /*self*/, PyObject* args)
 	PyDict_SetItemString(dict, "Usages2", PyLong_FromLong(si->Usages[2]));
 	PyDict_SetItemString(dict, "Flags", PyLong_FromLong(si->Flags));
 	PyDict_SetItemString(dict, "Header", PyLong_FromLong(header));
+	PyDict_SetItemString(dict, "Slot", PyLong_FromLong(idx));
+	PyDict_SetItemString(dict, "PC", PyLong_FromLong(globalID));
```

Comment:
Add these to the docs please. But it's also odd that you need PC at all (besides it being a misnomer) — how come?

Reply:
Its used here:
```
def OpenItemInfoWindow (slotItem):
	pc = slotItem["PC"]
	slotType = GemRB.GetSlotType (slotItem["Slot"], pc)
```

Reply:
In all three uses you have it available already, so could just pass it as a parameter and avoid this hoop.